server/scripts/backup.sh

```
@@ -151,3 +161,24 @@ restic forget --prune \
+# arrive on schedule. Any failure above aborts the script (set -e) before this
+# line, so a failed backup = no ping = an alert. See
+# docs/developer/operations.md for setup.
+
+OPS_ENV="$HOME/.config/loom-clone-ops.env"
+if [[ -f "$OPS_ENV" ]]; then
+  # shellcheck disable=SC1090
+  source "$OPS_ENV"
+fi
+if [[ -n "${HC_BACKUP_URL:-}" ]]; then
+  curl -fsS -m 10 --retry 5 "$HC_BACKUP_URL" > /dev/null \
+    && log "healthchecks ping sent" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)